### ISSUE TITLE
App drawer sideways

### ISSUE THREAD

**Issue by SchattengestaIt** [tdm/android_device_fxtec_pro1#59]
The app drawer is sideways which makes it quite unintuitive. Since it also occurrs in portrait mode, I don´t think this is a new "feature" of Android 9.

I wasn´t able to upload screenshots here so I posted the direct link on imgur.
https://i.imgur.com/qSue9VS.png - Landscape
https://i.imgur.com/J2jTe2R.png - Portrait

**Comment by enigma9o7** [comment]
I don't think you mean app drawer.  App drawer is where all the app icons are.

I think you mean recent apps?  But those for me display screenshot of when they were minimized.     I think all this is expected behavior, unless you're describing something else.

What exactly do you think is sideways or what exactly do you want different?

**Comment by SchattengestaIt** [comment]
Yes, I am talking about the recent apps. If you have the three icon menu buttons, you tap on the square to open the menu.
However, I found out that this is an Android 9 "feature" and it can only be solved with some launchers and specific apps for Samsung and OnePlus. It's a shame but you get used to it.
Sorry for bothering you, this can be closed.
